audioipc/src/rpccore.rs

```diff
@@ -178,12 +178,6 @@ pub(crate) fn make_client<C: Client>(
         in_flight: VecDeque::with_capacity(32),
     };
 
-    // Sender is Send, but !Sync, so it's not safe to move between threads
-    // without cloning it first.  Force a clone here, since we use Proxy in
-    // native code and it's possible to move it between threads without Rust's
-    // type system noticing.
-    #[allow(clippy::redundant_clone)]
-    let tx = tx.clone();
```

Comment:
Do we still have the issue mentioned in the comment?

Reply:
It's fixed by splitting the Proxies in ServerStreamCallbacks.  The clone added here was based on an incomplete understanding of the root cause of this bug.